src/components/users/profile/ko/runtime/profile.ts

```diff
@@ -143,7 +143,9 @@ export class Profile {
         if (isDelegationApplied) {
             return;
         }
-        const confirmed = window.confirm(`Dear ${this.user().firstName} ${this.user().lastName}, \nYou are about to close your account associated with email address ${this.user().email}.\nYou will not be able to sign in to or restore your closed account. Are you sure you want to close your account?`);
+        const confirmed = window.confirm(
+            this.isBasicAccount() ? `Dear ${this.user().firstName} ${this.user().lastName}, \nYou are about to close your account associated with email address ${this.user().email}.\nYou will not be able to sign in to or restore your closed account. Are you sure you want to close your account?`
+            : `Dear ${this.user().firstName} ${this.user().lastName}, \nYou are about to close your account associated with email address ${this.user().email}.\nAre you sure you want to close your account?`);
```

Comment:
Is there a possibility to move this text to some contants? 
It will be easier to mdify,change in future, and not change html file itself,
What do you think?

Reply:
as discussed, I extracted them in methods, thanks :)